programs/keeper-client/KeeperClient.cpp

```diff
@@ -43,8 +97,11 @@ String KeeperClient::executeFourLetterCommand(const String & command)
     return result;
 }
 
-std::vector<String> KeeperClient::getCompletions(const String & prefix) const
+std::vector<String> KeeperClient::getCompletions(String prefix) const
 {
+    /// Append trailing double quote to pass the parser correctly
+    if (!prefix.ends_with('"'))
+        prefix.append("\"");
```

Comment:
This won't allow autocompletion for commands and constants

ge + TAB -> get
create "foo" EPHE + TAB -> EPHEMERAL